File: gowe-server/gowe-module-system/gowe-module-system-start/src/main/java/com/github/alucardlockon/gowe/GoweSystemApplication.java
package com.github.alucardlockon.gowe;

import lombok.extern.slf4j.Slf4j;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
import org.springframework.context.ConfigurableApplicationContext;

/**
 * GOWE后台启动类
 */
@Slf4j
@SpringBootApplication
public class GoweSystemApplication extends SpringBootServletInitializer {

    public static void main(String[] args) {
        ConfigurableApplicationContext application = SpringApplication.run(GoweSystemApplication.class);
        log.info("gowe system is started!");
    }

}

File: gowe-server/gowe-module-system/gowe-module-system-start/src/main/kotlin/com/github/alucardlockon/gowe/GoweSystemApplication.kt
package com.github.alucardlockon.gowe

import lombok.extern.slf4j.Slf4j
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

/**
 * GOWE后台启动类
 */
@SpringBootApplication
@Slf4j
open class GoweSystemApplication

fun main(args: Array<String>) {
    runApplication<GoweSystemApplication>(*args)
}
